src/BuildIt.General/BuildIt.General.nuspec

```diff
@@ -18,7 +18,7 @@
       </group>
       <group targetFramework="UAP10.0.16299">
         <dependency id="Newtonsoft.Json" version="11.0.2" exclude="Build,Analyzers" />
-        <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.1.4" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.1.5" exclude="Build,Analyzers" />
```

Comment:
I'm wondering whether this reference should be removed from the nuspec?

Reply:
I was thinking about that too, but since a uwp app must have a reference to this, wouldn't it at least be helpful for this to "force" people to get the latest version? (What does this package even do?)